src/config/passport.conf.js

```diff
@@ -131,9 +133,14 @@ export default (passport) => {
         );
       }
 
+      // If the user's email address is not verified yet
+      if (!user.verified) {
+        return done(null, false, { loginMessage: 'Please confirm your email address. <a href="/verify">Send again</a>'})
```

Comment:
I thought I already fixed that... Must be an older version pushed to origin after I ran a rebase. Will try to fix it soon.